### PR TITLE
[Merged by Bors] - chore: rename invOf lemmas to match inv lemmas

### PR DESCRIPTION
**Pull request by eric-wieser**
This follows on from #11530

In each case, I've added an `example` with the `Group` lemma, to make it easier to keep the names in sync.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Algebra/Group/Invertible/Defs.lean [hunk 1]
```diff
@@ -90,51 +90,64 @@ class Invertible [Mul α] [One α] (a : α) : Type u where
...
 
 @[simp]
-theorem invOf_mul_self_assoc' [Monoid α] (a b : α) {_ : Invertible a} : ⅟ a * (a * b) = b := by
+theorem invOf_mul_cancel_left' [Monoid α] (a b : α) {_ : Invertible a} : ⅟ a * (a * b) = b := by
   rw [← mul_assoc, invOf_mul_self, one_mul]
+example {G} [Group G] (a b : G) : a⁻¹ * (a * b) = b := inv_mul_cancel_left a b
 
-theorem invOf_mul_self_assoc [Monoid α] (a b : α) [Invertible a] : ⅟ a * (a * b) = b := by
-  rw [← mul_assoc, invOf_mul_self, one_mul]
+theorem invOf_mul_cancel_left [Monoid α] (a b : α) {_ : Invertible a} : ⅟ a * (a * b) = b :=
```
Comment (YaelDillies) [inline Mathlib/Algebra/Group/Invertible/Defs.lean:113]: If I understand correctly the difference between the primed and non-primed lemmas, is this not supposed to be
```suggestion
theorem invOf_mul_cancel_left [Monoid α] (a b : α) [Invertible a] : ⅟ a * (a * b) = b :=
```
?

Comment (eric-wieser) [reply Mathlib/Algebra/Group/Invertible/Defs.lean]: Good catch

**Review by YaelDillies — APPROVED** [review]
Thanks for the followup!

maintainer merge

**Comment by kbuzzard** [comment]
Thanks!

bors merge

**Pull request closed without merging**